features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/des/des_alt_sw.h

```diff
@@ -145,7 +137,7 @@ int mbedtls_des_sw_setkey_dec( mbedtls_des_sw_context *ctx, const unsigned char
  * \return         0
  */
 int mbedtls_des3_sw_set2key_enc( mbedtls_des3_sw_context *ctx,
-                      const unsigned char key[MBEDTLS_DES_KEY_SIZE * 2] );
+                                 const unsigned char key[MBEDTLS_DES_KEY_SIZE * 2] );
 
```

Comment:
Could you please explain why ever file is duplicated in `./targets/TARGET_NUVOTON/TARGET_M480` and `./targets/TARGET_NUVOTON/TARGET_NUC472`? As far as I can tell is the same code. If this is the case, then each specific target just needs a different device file defining what accelerator is available in the board.

Reply:
@andresag01 Same as above.